### PR TITLE
creation of new activity enabling google user to sign up/sign

### PR DESCRIPTION
**Pull request by Gerussi**
creation of new activity enabling google user to sign up/sign google credentials, tests have yet to be made for this new activity

### CODE REVIEW

**Review by TruddyTheDuddi — APPROVED** [review]
Approved. Maybe some clean up in the future, remove update if not used

File: app/src/main/res/menu/menu_main.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:android="http://schemas.android.com/apk/res/android">
+
+    <item
+        android:id="@+id/miLogout"
+        android:title="Logout" />
+</menu>
```
[the remark this answers is not in the document]
Comment (Gerussi) [reply app/src/main/res/menu/menu_main.xml:8]: Yes, this was only a quick solution to test the logout functionality, the actually button could be relocated throughout the project if we change the layout of the app

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -9,4 +9,14 @@
 
     <string name="hello_first_fragment">Hello first fragment</string>
     <string name="hello_second_fragment">Hello second fragment. Arg: %1$s</string>
```
[the remark this answers is not in the document]
Comment (Gerussi) [reply app/src/main/res/values/strings.xml:12]: Yes, this is a good idea, thank you.

File: README.md [hunk 3]
```diff
@@ -4,4 +4,6 @@ All User Stories and features: https://docs.google.com/document/d/1tU0rLP7YBuYWO
 
 Git cheat sheet: https://education.github.com/git-cheat-sheet-education.pdf
 
-# Other
\ No newline at end of file
+# Other
+
+email: palfinder.sdp@gmail.com
```
[the remark this answers is not in the document]
Comment (Gerussi) [reply README.md:9]: Yep, done ;)

**Review by pgilliar — APPROVED** [review]

**Pull request merged**